src/sdk/clients/decorators/erc7579/erc7579.decorators.test.ts

```diff
@@ -31,13 +40,13 @@ describe("erc7579.decorators", async () => {
 
     chain = network.chain
     bundlerUrl = network.bundlerUrl
-    account = getTestAccount(0)
+    eoaAccount = getTestAccount(0)
```

Comment:
Replace account everywhere then, but I thought account was fine.

Reply:
account gets confused with smart account which is also named "account" in "nexusClient.account", will replace everywhere